### PR TITLE
Timos sap dot1q graphing fix

### PR DESCRIPTION
**Pull request by loopodoopo**
When a SAP is configured for dot1q and q-in-q '*' characters are used as the value for the OID. These characters can not be used in the file name for the RRD so we need to replace them to make it work. 

Not working:
image

Not working RRD command:

Note the location of the RRD and the '_' in the path replacing the '*'

```
rrdtool graph /tmp/laexPtRAqeKVcHJX  --alt-autoscale-max --rigid -E --start 1641314400 --end 1641400800 --width 1152 --height 652 -c BACK\#EEEEEE00 -c SHADEA\#EEEEEE00 -c SHADEB\#EEEEEE00 -c CANVAS\#FFFFFF00 -c GRID\#a5a5a5 -c MGRID\#FF9999 -c FRAME\#5e5e5e -c ARROW\#5e5e5e -R normal -c FONT\#000000 --font LEGEND:8:DejaVuSansMono --font AXIS:7:DejaVuSansMono --font-render-mode normal COMMENT:'SAP Traffic          Now      Min      Max     Avg\\l' DEF:ds0=10.40.35.30/sap-50000537.35913728._.rrd:sapIngressBits:AVERAGE DEF:ds0min=10.40.35.30/sap-50000537.35913728._.rrd:sapIngressBits:MIN DEF:ds0max=10.40.35.30/sap-50000537.35913728.**_**.rrd:sapIngressBits:MAX DEF:ds1=10.40.35.30/sap-50000537.35913728.**_**.rrd:sapEgressBits:AVERAGE DEF:ds1min=10.40.35.30/sap-50000537.35913728.**_**.rrd:sapEgressBits:MIN DEF:ds1max=10.40.35.30/sap-50000537.35913728.**_**.rrd:sapEgressBits:MAX DEF:ds2=10.40.35.30/sap-
<output ommitted> 
```

Working:
image

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: workspace.code-workspace [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (murrant) [inline workspace.code-workspace:1]: Can you drop the vscode config file?  Might be worth adding it to the .gitignore

Comment (loopodoopo) [reply workspace.code-workspace:1]: Yes. Sorry for that. Would you like me to add "*.code-workspace" to .gitignore in this PR as well?

Comment (PipoCanaja) [reply workspace.code-workspace:1]: @loopodoopo Would be better to remove the file only in this PR , and create a new PR to change the gitfile. Just for readability. Thx

Comment (loopodoopo) [reply workspace.code-workspace]: Sure, will do!

**Comment by PipoCanaja** [comment]
@loopodoopo 
When looking at the files you change , and at the description of this PR, I don't really get it. Are you sure this PR contains all the relevant files and changes ?

**Comment by loopodoopo** [comment]
Clarified PR as per @PipoCanaja's suggestions in PM.

**Review by PipoCanaja — APPROVED** [review]
LGTM

**Pull request merged**